src/cpp/session/modules/SessionConnectionsInstaller.R

```diff
@@ -333,12 +333,26 @@
    prereqs()
 })
 
-.rs.addFunction("odbcBundleOdbcinstPath", function() {
+.rs.addFunction("odbcBundleOdbcinstPathWithOdbcinst", function() {
    config <- system2("odbcinst", "-j", stdout = TRUE)
    odbciniEntry <- config[grepl("odbcinst.ini", config)]
    gsub("^[^/\\\\]*", "", odbciniEntry)
 })
 
+.rs.addFunction("odbcBundleOdbcinstPathUseHome", function() {
+   normalizePath("~/odbcinst.ini", mustWork = FALSE)
```

Comment:
```suggestion
   normalizePath("~/.odbcinst.ini", mustWork = FALSE)
```

Reply:
Thanks! James confirmed this works in his environment as well.